Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM osawagiboy/slimane-docker:latest
```

Comment:
OK🙃Fixed.
I created [slimane team] (https://hub.docker.com/u/slimane/dashboard/) and invite you.